frontend/src/scenes/web-analytics/webAnalyticsLogic.ts

```diff
@@ -734,12 +745,54 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                 return !!geoIpPluginConfig && geoIpPluginConfig.enabled
             },
         },
+        initialFilters: {
+            _default: null as boolean | null,
+            loadInitialFilters: async (): Promise<WebAnalyticsPropertyFilters> => {
+                // This code guesses which is the marketing domain based on whether it's www or apex. We use this to set
+                // the $host filter's initial state. Later on we'll have this as config that people can set, but for now
+                // we try to do the smart thing without people needing to do anything
+                const params = new URLSearchParams([
+                    ['key', '$host'],
+                    ['event_name', '$pageview'],
+                    ['event_name', '$pageleave'],
+                    ['event_name', '$autocapture'],
+                ])
+                let propertiesResponse: unknown
+                try {
+                    propertiesResponse = await api.get(`api/event/values/?${params}`)
+                } catch {
+                    // ignore
+                }
+                if (!propertiesResponse || !Array.isArray(propertiesResponse) || propertiesResponse.length <= 1) {
+                    return []
+                }
+                const hosts: string[] = propertiesResponse.map((x) => x.name)
+                // at this point we have multiple hosts, pick the ones that are www or apex
+                const wwwOrApex = hosts.filter((host) => {
+                    const subdomain = tldts.getSubdomain(host)
+                    return subdomain === '' || subdomain === 'www'
+                })
```

Comment:
Perhaps these subdomains on their own hold value? Suggestion that's non blocking: rename this to a `subdomains` loader, and use a selector to derive the filters.